### DISCUSSION TITLE
Did the install correctly but still somehow doesn't work....please help

### DISCUSSION THREAD

**Discussion by abidurX** [lllyasviel/Fooocus#3253] [Q&A] [answered]
Did everything to the instructions and ran the run.bat and the software doesn't work in two ways.
1. after running run.bat, the browser opens and shows a blank window with no interface  of the software whatsoever. Sometimes it doesn't even get to step of browser and browser crashes. (attached some screenshots to better understand)

![Screenshot 2024-07-08 203303](https://github.com/lllyasviel/Fooocus/assets/143009924/8ead1c9d-2b5e-4e3c-a9a7-480585aeed9a)
![Screenshot 2024-07-08 203333](https://github.com/lllyasviel/Fooocus/assets/143009924/9e4bb6da-7e73-495b-b567-04e0dfc8bf7d)
![Screenshot 2024-07-08 203333](https://github.com/lllyasviel/Fooocus/assets/143009924/70a4d94b-b8d9-4e05-a1f5-67ddb71af694)

2. after running run.bat, some error window pops up. (see the screenshots below.)
![WhatsApp Image 2024-07-08 at 20 40 30_68932fa1](https://github.com/lllyasviel/Fooocus/assets/143009924/61c103cc-5266-4dea-a11d-26ca269fee87)
![WhatsApp Image 2024-07-08 at 20 40 30_d737af70](https://github.com/lllyasviel/Fooocus/assets/143009924/565fbbed-58cb-41c7-bbc7-73d22ee8b0cf)
![WhatsApp Image 2024-07-08 at 20 40 29_0f02ca05](https://github.com/lllyasviel/Fooocus/assets/143009924/aba94251-5bea-4a9b-8742-a7397588b1f4)
![WhatsApp Image 2024-07-08 at 20 40 29_da588972](https://github.com/lllyasviel/Fooocus/assets/143009924/7d1d5394-cbc0-40c2-a9fd-d7394089f823)

any help to solve this will be greatly appreciated.....

**Comment by ILikeToasters** [comment, marked as answer]
Have you checked here? Most common problem is usually System Swap: https://github.com/lllyasviel/Fooocus/blob/5a71495822a11bbabf7c889eed6d9b38b261bb96/troubleshoot.md#L4
